envoy/config/filter/http/jwt_authn/v2/BUILD

```diff
@@ -0,0 +1,9 @@
+licenses(["notice"])  # Apache 2
+
+load("//bazel:api_build_system.bzl", "api_proto_library")
+
+api_proto_library(
+    name = "jwt_authn",
```

Comment:
jwt_authz .. not authn

Reply:
it should be authn.   "authz" is for authorization.  This proxy is only for token verification, it is authentication, it is "authn".